### ISSUE TITLE
Тема доклада: HTML

### ISSUE THREAD

**Issue by SergeyMirvoda** [ownagesan/WEB-programming#2]
- История создания HTML - на пару минут (когда, кем и для чего)
- Структура HTML документа (html\head\body\h1\p) и т.д
- Формы (кнопки, поля ввода)
- Версии HTML (не включая HTML5) - тоже на пару минут.

**Comment by ownagesan** [comment]
HTML

**Comment by SergeyMirvoda** [comment]
Обязательно подготовить доклад и быть на лекции.
Ваш доклад должен быть первым

**Comment by ownagesan** [comment]
Хорошо

**Comment by Slevel** [comment]
Доклад необходимо подготовить к следующей лекции, или прикрепить тут заранее?

**Comment by SergeyMirvoda** [comment]
Лучше сюда, файлом, заранее.

**Comment by Slevel** [comment]
Будет сделано =)

**Comment by ownagesan** [comment]
![prohtml](https://cloud.githubusercontent.com/assets/6747970/6326726/82e283a4-bb76-11e4-8ffa-41c34b53504f.png)

**Comment by ownagesan** [comment]
https://yadi.sk/d/ZEaIoPqmeqjGZ
Там HTML файл с докладом

**Comment by SergeyMirvoda** [comment]
> Там HTML файл с докладом

Отлично!
Но -- нужно **положить файлы в свой репозиторий**
и привести здесь ссылку.

**Comment by ownagesan** [comment]
https://github.com/ownagesan/WEB-programming.git

**Comment by ownagesan** [comment]
https://github.com/ownagesan/WEB-programming/blob/master/ProHTML.html

**Comment by SergeyMirvoda** [comment]
Вставьте в нужное место этот тег. Иначе Не 
Смог прочитать. 

``` html
<meta http-equiv="content-type" content="text/html; charset=utf-8">
```

**Comment by ownagesan** [comment]
Добавил

**Comment by SergeyMirvoda** [comment]
Чего то не помогло. Там точно уникод?
http://htmlpreview.github.io/?https://github.com/ownagesan/WEB-programming/blob/master/ProHTML.html

**Comment by ownagesan** [comment]
У меня без тега meta все читалось, а с ним нет, сейчас посмотрю

**Comment by ownagesan** [comment]
Вот так должно быть?
![2015-02-24 15-09-40](https://cloud.githubusercontent.com/assets/6747970/6347170/9e6cfe76-bc37-11e4-8bc3-a6cbe36a3bf5.png)

**Comment by SergeyMirvoda** [comment]
У меня телефон. Там хром не такой умный. Не умеет автоматически кодировку определять. 
Ладно доберусь до нормального браузера. Прочитаю.

**Comment by SergeyMirvoda** [comment]
![screenshot_2015-02-24-15-17-36](https://cloud.githubusercontent.com/assets/286948/6347264/4f8f6374-bc38-11e4-995d-5e008b08eeb8.png)

**Comment by SergeyMirvoda** [comment]
Видимо файл не в utf-8 у вас.

**Comment by ownagesan** [comment]
Да, я понял, просто сохранял по умолчанию, забыл про кодировку, сейчас попробуйте

**Comment by SergeyMirvoda** [comment]
Отлично. Заработало.

**Comment by SergeyMirvoda** [comment]
Все хорошо. 
Напишите ещё в разделе версий про xhtml и doctype

**Comment by ownagesan** [comment]
Хорошо, а доклад рассказывать будем по этому файлу или можно с вордовского, там где изначально была собрана информация?

**Comment by SergeyMirvoda** [comment]
Как вам удобнее. 
Еще забыл. Список источников нужен. Факты

**Comment by ownagesan** [comment]
Попозже добавлю все в репозиторий

**Comment by ownagesan** [comment]
Добавлено про XHTML и DOCTYPE, список источников в файле "Сайты" в репозитории.

**Comment by Slevel** [comment]
Сергей, позвольте уточнить, доклад будет рассказан нами завтра, 25.02.15? 
Среди ребят в группе мнения разделились, к какой же паре готовить работы, не хочется попасть в просак, но и зря готовиться тоже =)

**Comment by SergeyMirvoda** [comment]
Тем кому можно позже я персонально каждому написал.  
Те кто не выложил работу могут уже не готовиться.

**Comment by Slevel** [comment]
Хорошо)

**Comment by ownagesan** [comment]
Алешкович Александр, РИ-320401
